Octokit/Models/Request/SearchRepositoriesRequest.cs

```diff
@@ -395,355 +395,677 @@ public enum Language
     {
 #pragma warning disable 1591
         [SuppressMessage("Microsoft.Naming", "CA1704:IdentifiersShouldBeSpelledCorrectly", MessageId = "Abap")]
+        [Parameter(Value = "abap")]
         Abap,
+
         [Parameter(Value = "ActionScript")]
         ActionScript,
+        
+        [Parameter(Value = "ada")]
         Ada,
+        
+        [Parameter(Value = "apex")]
         Apex,
+        
         [Parameter(Value = "AppleScript")]
```

Comment:
It also seems like we could change Repository.Language and Gist.Language to be the proper `StringEnum<Language>` instead on just `string`, if we wanted to